jdaviz/core/registries.py

```diff
@@ -115,12 +115,12 @@ def decorator(cls):
                     f"registered components must inherit from "
                     f"`ipyvuetify.VuetifyTemplate`.")
 
-            self.add(name, cls, label, icon, viewer_requirements, overwrite)
+            self.add(name, cls, label, icon, viewer_requirements, category, overwrite)
             return cls
         return decorator
 
     def add(self, name, cls, label=None, icon=None,
-            viewer_requirements=[], overwrite=False):
+            viewer_requirements=[], category=None, overwrite=False):
```

Comment:
It seems like `viewer_requirements` was removed from all of the plugins, should it be removed from here as well? Or is it still used by something else?

Reply:
Good point - removed the rest of the instances and we'll see if any tests complain 🤞